jenkins/job-configs/kubernetes-jenkins/kubernetes-test-go.yaml

```diff
@@ -11,6 +11,8 @@
         - shell: |
             export KUBE_FORCE_VERIFY_CHECKS='y'
             export KUBE_VERIFY_GIT_BRANCH='{branch}'
+            export KUBE_TEST_API_VERSIONS='v1,extensions/v1beta1,metrics/v1alpha1,federation/v1alpha1;v1,autoscaling/v1,batch/v1,batch/v2alpha1,extensions/v1beta1,apps/v1alpha1,metrics/v1alpha1,federation/v1alpha1,policy/v1alpha1,rbac.authorization.k8s.io/v1alpha1'
```

Comment:
I'm not sure we want to have this list in two places (here and hack/test-go.sh) - we're almost certainly going to forget to update one of them.